client/signup/steps/plans-features/index.jsx

```diff
@@ -0,0 +1,102 @@
+/**
+ * External dependencies
+ */
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { localize } from 'i18n-calypso';
+import isEmpty from 'lodash/isEmpty';
+
+/**
+ * Internal dependencies
+ */
+import PlansFeaturesMain from 'my-sites/plans-features-main';
+import StepWrapper from 'signup/step-wrapper';
+import SignupActions from 'lib/signup/actions';
+import QueryPlans from 'components/data/query-plans';
+import { getPlans } from 'state/plans/selectors';
+
+class PlansFeaturesStep extends Component {
+	constructor( props ) {
+		super( props );
+
+		this.onUpgradeClick = this.onUpgradeClick.bind( this );
+	}
+
+	plansList() {
+		const {	plans, hideFreePlan } = this.props;
+
+		return (
+			<div>
+				<QueryPlans />
+
+				<PlansFeaturesMain
+					plans={ plans }
+					site={ {} }
```

Comment:
So that using `site.jetpack` does't throw `undefined` error. But we can just specify `site` in `defaultProps` to be an empty obj.